ts/services/audioRecorder.ts

```diff
@@ -113,15 +138,7 @@ export class RecorderClass {
 
     this.clear();
 
-    if (error && error.name === 'NotAllowedError') {
-      log.warn('Recorder/onError: Microphone permission missing');
-      window.showPermissionsPopup();
-    } else {
-      log.error(
-        'Recorder/onError:',
-        error && error.stack ? error.stack : error
-      );
-    }
+    log.error('Recorder/onError:', error && error.stack ? error.stack : error);
```

Comment:
I noticed when looking at logs the other days that this logs an empty object. Would be super awesome to fix this in this PR, but if you want to defer it that's understandable.